controllers/slicegateway/reconciler.go

```diff
@@ -144,7 +144,7 @@ func (r *SliceGwReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	// true if the gateway is openvpn server
 	// Check if the Gw service already exists, if not create a new one if it is a server
 	if isServer(sliceGw) {
-		reconcile, result, err = r.handleSliceGwSvcCreation(ctx, sliceGw, sliceGwNodePort)
+		reconcile, result, sliceGwNodePort ,err = r.handleSliceGwSvcCreation(ctx, sliceGw)
```

Comment:
Please fix indentation around the ','

reconcile, result, sliceGwNodePort, err

Applies to the rest of the diff as well. There should be a space between the comma and the next var, but not between a var and a comma.

Please run go fmt.

Reply:
done